docs/Containerization.md

```diff
@@ -123,19 +124,6 @@ The frontend would only be available after the containers are running.
 
 Locally, open a web browser and go to [http://localhost:3000](http://localhost:3000).
 
-### Sample Accounts
-
-By default, we have created 2 accounts as part of the fresh database. These accounts are for testing purposes, and would not be avilable for production.
-
-| Email          | Password | Remarks                                  |
-| -------------- | -------- | ---------------------------------------- |
-| admin@test.com | password | Has admin rights. Regard as a superuser. |
-| user@test.com  | password | Normal user. Cannot manage other users.  |
-
-> Note: if you delete these accounts via the user profile management
-> functionalities, you may have from restart the pre-testing steps to
-> get back the accounts.
-
 ## Post-testing
```

Comment:
I think a sample admin account is still necessary? Else I dont think there is any where a tester/anyone that is not admin could try out admin accounts

Reply:
Added new docs regarding test accounts